src/components/MissionList.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Table from 'react-bootstrap/Table';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+import { useDispatch } from 'react-redux';
+import { reserveMission, leaveMission } from '../redux/missions/missionsSlice';
+import '../styles/MissionList.css';
+
+const MissionList = ({ missions }) => {
+  const dispatch = useDispatch();
+
+  const handleJoinMission = (missionId) => {
```

Comment:
- [ ] From the missionList.jsx youA have not import the file fromthe components kindly fix it to make your page render and visible it display not on the screen when i clone it severally ,please kindly fix it.